DuckDuckGo/OnboardingExperiment/ContextualDaxDialogs/ContextualOnboardingList.swift

```diff
@@ -31,21 +31,22 @@ public enum ContextualOnboardingListItem: Equatable {
         case .search(let title):
             return title
         case .site(let title):
-            return title
+            return title.replacingOccurrences(of: "https:", with: "")
         case .surprise:
             return "Surprise me"
         }
     }
 
     var title: String {
-        let scheme = "https:"
         switch self {
         case .search(let title):
-            return scheme + title
+            return title
+                .replacingOccurrences(of: "”", with: "")
```

Comment:
interesting that we have to do twice 😓

Reply:
I think technically they are not the same here they look the same but actually are two different symbols  😅